library/Booster/CLOptions.hs

```diff
@@ -103,8 +103,16 @@ clOptionsParser =
             . fromKebab
 
 -- custom log levels that can be selected
-allowedLogLevels :: [String]
-allowedLogLevels = ["Rewrite", "Simplify", "Depth"]
+allowedLogLevels :: [(String, String)]
+allowedLogLevels =
+    [ ("Rewrite", "Log all rewriting in booster")
+    , ("RewriteKore", "Log all rewriting in kore-rpc fall-backs")
+    , ("RewriteSuccess", "Log successful rewrites (booster and kore-rpc)")
+    , ("Simplify", "Log all simplification/evaluation in booster")
+    , ("SimplifyKore", "Log all simplification in kore-rpc")
+    , ("SimplifySuccess", "Log successful simplifications (booster and kore-rpc)")
+    , ("Depth", "Log the current depth of the state")
+    ]
```

Comment:
I did not find a good way to display the information strings for the log levels in a table... (`optparse-applicative` reformats the help string) but still left them here for a follow-up